app/lib/image_resizer.rb

```diff
@@ -0,0 +1,22 @@
+module ImageResizer
+  DEFAULT_CL_OPTIONS = {
+    type: "fetch",
+    height: nil,
+    width: nil,
+    crop: "limit",
+    quality: "auto",
+    flags: "progressive",
+    fetch_format: "auto",
+    sign_url: true
+  }.freeze
+
+  def self.call(img_src, **kwargs)
```

Comment:
should we make it explicit that this requires an `image_url` ? Otherwise one might be tempted to pass a path to an asset as well

Reply:
Oh, I did not consider this. I will make it raise an exception.